noop/index.js

```diff
@@ -0,0 +1,43 @@
+const {default: traverse} = require("@babel/traverse");
+const {looksLike} = require("../utils/looks-like");
+
+
+module.exports = () => {
+	return {
+		name: "willMutate",
+		visitor: {
+			/**
+             * We need to traverse the "Program" so we can know if we need to inject
+             * the Proxy handler or not.
```

Comment:
Nit: spaces

Reply:
Legit no clue how this happened

Reply:
ESLint doesn't track `indent` on jsdoc-styled block comments IIRC.